html-css/week1/mateus_britto/order-desktop/styles.css

```
@@ -0,0 +1,344 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif, sans-serif;
+  background-color: #808080;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  color: #ffffff;
+  margin: 30px;
+  font-family: "Montserrat", serif;
+  padding: 20px;
+}
+
+.header-title {
+  text-align: center;
+  font-size: 2rem;
+  margin-bottom: 10px;
+}
+
+.header-subtitle {
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.sections-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 800px;
+  height: auto;
+  background-color: #ffffff;
+}
+
+.order-summary {
+  background: linear-gradient(90deg, #fd7b46, #fe4e9b);
+  color: #ffffff;
+  width: 100%;
+  height: auto;
+  padding: 40px 20px 20px 20px;
+}
+
+.total-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.order-summary-title {
+  margin-bottom: 10px;
+}
+
+.total-top {
+  border-bottom: solid 1px #ffffff;
+}
+
+.total-top > p {
+  font-size: 18px;
+  margin-bottom: 5px;
+}
+.total-bottom > p {
+  font-size: 18px;
+  font-weight: bold;
+  margin-top: 5px;
+}
+
+.payment-method {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 171px;
+  margin-top: 30px;
+}
+
+.payment-method-title {
+  font-size: 24px;
+}
+
+.buttons-div {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 20px;
+  width: 100%;
+  padding: 20px;
+}
+
+.payment-method-button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  cursor: pointer;
+  width: 100%;
+  height: 110px;
+  background-color: #ffffff;
+  font-size: 1.125rem;
+  font-family: "Montserrat", serif;
+  font-weight: bold;
+  font-style: italic;
+  color: #939393;
+  border-radius: 10px;
+  border: none;
```

Set the border already on the default state, but have the color as `transparent`. This will prevent the jumping you have now on hover.
```suggestion
  border: solid 4px transparent;
```